apps/src/templates/sessions/LockoutPanel.tsx

```diff
@@ -0,0 +1,466 @@
+              ...dateOptions,
+              timeZone: 'UTC',
+            }),
+          })}
+        </p>
+
+        {/* This field shows the current status of the validation. */}
+        {/* Parent Permission: Not Pending / Pending */}
+        <div style={styles.statusSection}>
+          <div>
+            <label
+              style={isRTL ? styles.statusLabelRTL : styles.statusLabel}
+              htmlFor="permission-status"
+            >
+              <strong>{i18n.sessionLockoutParentStatusField()}</strong>
+            </label>
+            <span id="permission-status" style={permissionStatus.style}>
+              <strong>{permissionStatus.message}</strong>
+            </span>
+          </div>
+
+          {/* This is a floating 'link' that resends the pending email. */}
+          {pendingEmail &&
+            status !== ChildAccountComplianceStates.PERMISSION_GRANTED && (
+              <Button
+                id="lockout-resend"
+                styleAsText={true}
+                style={styles.resendLink}
+                text={i18n.sessionLockoutResendEmail()}
+                type="button"
+                onClick={resendPermissionEmail}
+              />
+            )}
+        </div>
+
+        {/* This field allows the input of an email address. */}
+        {/* Parent Email: [email] */}
+        {status !== ChildAccountComplianceStates.PERMISSION_GRANTED && (
+          <div style={styles.sections}>
+            <div style={styles.section}>
+              <label
+                style={isRTL ? styles.labelRTL : styles.label}
+                htmlFor="parent-email"
+              >
+                <strong>{i18n.sessionLockoutParentEmailField()}</strong>
+              </label>
+
+              {/* Slightly complicated layout allows for text underneath. */}
+              <div style={styles.fieldSection}>
+                <input
+                  style={styles.field}
+                  onChange={onEmailUpdate}
+                  onInput={onEmailUpdate}
+                  onBlur={onEmailUpdate}
+                  defaultValue={parentEmail}
+                  name="parent-email"
+                  id="parent-email"
+                />
+
+                {/* Show a 'Last email sent' prompt when available. */}
+                {pendingEmail && lastEmailDate && (
+                  <p style={styles.lastEmail}>
+                    <em id="lockout-last-email-date">
+                      {i18n.sessionLockoutLastEmailSent() + ' '}
+                      {lastEmailDate.toLocaleDateString(locale, {
+                        ...dateOptions,
+                        hour: 'numeric',
+                        minute: 'numeric',
+                      })}
+                    </em>
+                  </p>
+                )}
+              </div>
+            </div>
+          </div>
+        )}
+
+        {status !== ChildAccountComplianceStates.PERMISSION_GRANTED && (
```

Comment:
This condition is the same as the one directly above it. That seems correct though redundant. Is this just to avoid the indent? Maybe we can pull out the form rendering to its own function.